src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
The name of the interface 'AreaCalculator' suggests an action, which is not recommended for interfaces. Consider renaming it to something that represents an entity or a capability like 'AreaComputable' or 'MeasurableArea'.